tests/test_inspection.py

```diff
@@ -2,28 +2,31 @@
 from pydantic import ValidationError
 from pipeline import FertilizerInspection
 
-class TestFertiliserForm(unittest.TestCase):
-    def test_valid_fertiliser_form(self):
-        data = {
+class TestFertiliserInspection(unittest.TestCase):
+    def setUp(self):
+        self.data = {
             "company_name": "ABC Company",
             "company_address": "123 Main St",
             "fertiliser_name": "Super Fertiliser",
-            "npk": "10-5-5",
```

Comment:
?

Reply:
Because of this https://github.com/ai-cfia/fertiscan-pipeline/blob/d66a51b45aa4863f650fc69528a3a8b1b36ac699/tests/test_inspection.py#L39

Just to avoid a conflict with the new npk